rust/agents/relayer/src/msg/op_queue.rs

```diff
@@ -98,8 +105,12 @@ impl OpQueue {
     /// Get the metric associated with this operation
     fn get_operation_metric(&self, operation: &dyn PendingOperation) -> IntGauge {
         let (destination, app_context) = operation.get_operation_labels();
-        self.metrics
-            .with_label_values(&[&destination, &self.queue_metrics_label, &app_context])
+        self.metrics.with_label_values(&[
+            &destination,
+            &self.queue_metrics_label,
+            &operation.status().to_string(),
```

Comment:
very clean that this is pretty much the only change to plug things together w/ updated metric labels